portfolio/src/main/webapp/index.html

```diff
@@ -14,10 +14,15 @@ <h1>Heidi Johnson's Portfolio</h1>
             about me by browsing this site!</p>
             <p>Click the links below to learn about my background, 
                 browse some of my photos, find out random facts, and more :)</p>
-            <a class="link" href="aboutme.html">About Me</a>
-            <a class="link" href="gallery.html">Gallery</a>
-            <a class="link" href="random.html">Random Facts</a>
-            <a class="link" href="planet.html">Sustainability</a>
+            <div class="logo">
+              <img id="sunshine" src="images/sunshine.png"/>
+              <div class="links">
+                <a class="link" href="aboutme.html">About Me</a>
+                <a class="link" href="gallery.html">Gallery</a>
+                <a class="link" href="random.html">Random Facts</a>
+                <a class="link" href="planet.html">Sustainability</a>
+              </div>
+            </div>
```

Comment:
Please fix the indentation for the contents of the \<section> block to keep it consistent with the rest of your code

Reply:
Thank you for pointing this out! I fixed some indentations later on after being made aware of the inconsistency, and just fixed it here.